examples_tests/atari/test_collect_demos.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -Ceu
+
+outdir=$(mktemp -d)
+
+gpu="$1"
+
+# Chainer 4 does not support open_pickle_dataset_writer, which is used by the demonstration collection example.
+pickle_writer_support=$(python -c "import chainer; from distutils.version import StrictVersion; print(1 if StrictVersion(chainer.__version__) >= StrictVersion('5.0.0') else 0)")
```

Comment:
On some environment this line may not work because users might use `python3` instead of `python`.  
I think now this line can be removed because we no longer support Chainer V4.

Reply:
I don't think the Chainer v4 support has been dropped yet. That will be done separately in a different PR.

Regarding your other comment, I'm following this: https://github.com/chainer/chainerrl/pull/204/files#diff-875175ed9e296e37b03562cf27435d32R11.

And wouldn't your `python2` vs. `python3` comment apply to all example tests? E.g. https://github.com/chainer/chainerrl/blob/master/examples_tests/atari/test_a2c.sh

Reply:
Ah you're completely right.